### PR TITLE
Add ansible role for AWS

### PR DESCRIPTION
**Pull request by avery-blanchard**
Add ansible role for AWS. This role is not ready for use. The ansible AWS module currently does not have the functionality to create an AMI with UEFI and TPM enabled. An [issue](https://github.com/ansible-collections/amazon.aws/issues/944) has been opened regarding this. Next steps and potential blockers are addressed in the README for when the development of the playbook can resume. @lkatalin

### CODE REVIEW

File: keylime-ansible-aws/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+`$ pip3 install boto3 botocore ` 
+2. Install amazon.aws module for ansible \
+`$ ansible-galaxy collection install amazon.aws `
+3. Create and record the access key ID and secret key for your AWS account
+   1. Navigate to the 'IAM' section of the AWS console
+   2. On the IAM dashboard, select 'Users'
+   3. Select your user
+   4. Select 'Security credentials'
+   5. Click 'Create access key'
+   6. Record your Access key ID and secret key for authenticating the playbook
```
Comment (lkatalin) [inline keylime-ansible-aws/README.md]: I would add something like "Your access key ID is displayed in the table in the AWS console. Your secret key is the key itself." I think the first 3 times I read this, I thought they were two different keys.

File: keylime-ansible-aws/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+4. The playbook will generate a SSH key pair named 'ansible', add this path to an ansible configuration file in the current directory (`ansible.cfg`)
+```
+[defaults]
+private_key_file=~/.ssh/ansible.pem
+```
+
+5. Run script to configure environment for ansible. 
+```
+./set_env_var.sh --help
+"Usage: ./set_env_var.sh <AWS access key ID> <AWS secret key> <AWS region>"
```
Comment (lkatalin) [inline keylime-ansible-aws/README.md]: Is the AWS secret key supposed to be a file path to your key? Or what format does it expect here?

**Review by lkatalin — CHANGES_REQUESTED** [review]
A few small nits, otherwise my inclination is to go ahead and merge it. It can serve as a placeholder until the AWS ansible module PR comes through and/or until we upload the manual setup instructions separately.

File: keylime-ansible-aws/README.md (hunk 1 above)
Comment (avery-blanchard) [reply keylime-ansible-aws/README.md]: The secret key is also displayed in the console (only when first created), and is not the .csv file downloaded. I will add more clarity on the keys!

File: keylime-ansible-aws/README.md (hunk 2 above)
Comment (avery-blanchard) [reply keylime-ansible-aws/README.md]: If you can think of a better way to handle this, let me know. The two keys required to authenticate ansible are just displayed in the console when the access key is created. The .csv contains another value that does not match either the value for the secret key nor the access key ID. Currently, the secret key and access key ID are to be copied by the user when it is created then passed into the script.

Comment (lkatalin) [reply keylime-ansible-aws/README.md]: This makes more sense now

File: keylime-ansible-aws/README.md (hunk 1 above)
Comment (lkatalin) [reply keylime-ansible-aws/README.md]: I think this is fine

**Review by lkatalin — APPROVED** [review]
Thanks! The .csv file is parsed correctly by your env vars script. We can troubleshoot the actual automation later and/or add some manual instructions as a placeholder.

**Comment by lkatalin** [comment]
@mpeters What do you think, can we merge?

**Pull request merged**